nvflare/private/fed/server/server_engine.py

```diff
@@ -362,7 +361,7 @@ def abort_app_on_server(self, job_id: str) -> str:
                 self.run_processes.pop(job_id)
 
```

Comment:
This needs to be changed to:

        finally:
            with self.lock:
                if job_id in self.run_processes:
                    self.run_processes.pop(job_id)

Reply:
thanks, fixed.